composer.json

```
@@ -8,18 +8,21 @@
     "scripts": {
         "test": "vendor/bin/phpunit --configuration phpunit.xml < tests/input.txt",
         "lint": "vendor/bin/phpcs",
-        "format": "vendor/bin/phpcbf"
+        "format": "vendor/bin/phpcbf",
+        "check" : "./vendor/bin/phpstan analyse --level max src tests"
```

Let's be consistent:

```suggestion
        "check" : "vendor/bin/phpstan analyse --level max src tests"
```